index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
More vivid and contrasting colors. The earth tones too passive.. Let me know when you have competed. Keep up the good work.!! This is a nice piece of work.